decko-rails/spec/controllers/card_controller_spec.rb

```diff
@@ -5,8 +5,13 @@
   include Card::Location
   include Capybara::DSL
   describe "- route generation" do
+    routes { Decko::Engine.routes }
```

Comment:
This was a puzzle for a while.  Tests the use routes seem to need this or they won't match routes in the engine.